config/sc-config.yaml

```diff
@@ -1078,15 +1078,15 @@ fluentd:
           memory: 100Mi
         limits:
           cpu: 1000m
-          memory: 400Mi
+          memory: 2Gi
```

Comment:
I believe so, because at some points when it's downloading/expanding chunks that are 500Mb or more it goes over 1Gb of memory. (It OOMed a lot before bumping that)

Reply:
Is there a way to control the size of those chunks? Or the compression level?